### PR TITLE
[14.0][ADD] mrp_component_operation

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
This module allows to operate the components from a MO, being able to move it to another location or making an scrap
and take another from a desired location and link it to the current MO.

@ForgeFlow

### CODE REVIEW

File: mrp_component_operation/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "MRP Components Operations",
+    "version": "14.0.1.1.0",
```
Comment (MiquelRForgeFlow) [inline mrp_component_operation/__manifest__.py]:
```suggestion
    "version": "14.0.1.0.0",
```

File: mrp_component_operation/models/mrp_production.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 ForgeFlow S.L. (https://www.forgeflow.com)
+# - Héctor Villarreal <hector.villarreal@forgeflow.com>
```
Comment (MiquelRForgeFlow) [inline mrp_component_operation/models/mrp_production.py]:
```suggestion
# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
# - Héctor Villarreal <hector.villarreal@forgeflow.com>
```
Héctor 😘

Comment (DavidJForgeFlow) [reply mrp_component_operation/models/mrp_production.py]: Sorry hector you have to go :cry:

File: mrp_component_operation/__manifest__.py (hunk 1 above)
Comment (DavidJForgeFlow) [reply mrp_component_operation/__manifest__.py]: Done thanks!

File: mrp_component_operation/views/mrp_component_operation_views.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<odoo>
```
Comment (MiquelRForgeFlow) [inline mrp_component_operation/views/mrp_component_operation_views.xml:2]:
```suggestion
<?xml version="1.0" encoding="utf-8" ?>
<odoo>
```
also in the other files

Comment (DavidJForgeFlow) [reply mrp_component_operation/views/mrp_component_operation_views.xml:2]: :+1:

**Comment by JuanyDForgeflow** [comment]
Functional review LGTM, are you going to add tests?

**Review by JoanSForgeFlow — APPROVED** [review]
LGTM :+1:

**Comment by DavidJForgeFlow** [comment]
> Functional review LGTM, are you going to add tests?

Currently added!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**